src/cards/ha-weather-card.html

```diff
@@ -14,23 +14,29 @@
         }
 
         .attribution {
-          color:grey;
+          color: var(--secondary-text-color);
+          text-align: right;
```

Comment:
So one of these is the weather card, the other is the more-info. The common parent is pretty much going to apply to all of the frontend, and the more I think about it, since these are very different UI elements they should be different anyway. They happen to be the same now, but they could easily need to be different in the future. I'm good with it as-is if you are.